tests/rust/forward_declaration.toml

```diff
@@ -1,23 +1,19 @@
-header = """
-#if 0
-''' '
-#endif
+header = [
```

Comment:
This doesn't seem particularly an improvement fwiw. But after this patch, is there any test that remains with `header = "string"`? Does that still work with this change?

Reply:
Yes, you can still use strings.

Reply:
Searching the codebase I found 1 test (swift_name) with `header = "string"` and 13 tests (asserted_cast/cfg_2/cfg/custom_header/deprecated/destructor_and_copy_ctor/layout_aligned_opaque/layout_packed_opaque/layout/must_use/nonnull_attribute/mod_attr/rename_crate) with
```
header = """
multi-line string
"""
```